src/Command/ColdStorageFlagCommand.php

```diff
@@ -66,8 +57,12 @@ public function __construct(EntityManagerInterface $entityManager, MessageBusInt
     protected function configure()
     {
         $this
-            ->setDescription('Marks dataset in input file is cold-stored and updates datafile')
-            ->addArgument('infofile', InputArgument::REQUIRED, 'Filename of the coldinfo-formatted textfile required.');
+            ->setDescription('Marks specified dataset as cold-stored and updates fileset with provided files')
+            ->addOption('udi', null, InputOption::VALUE_REQUIRED, 'UDI of dataset to flag as cold stored')
+            ->addOption('originalfilesize', null, InputOption::VALUE_REQUIRED, 'Original file size')
+            ->addOption('originalfilehash', null, InputOption::VALUE_REQUIRED, 'Original sha256 hash')
+            ->addOption('originalfilename', null, InputOption::VALUE_REQUIRED, 'Original file name')
```

Comment:
No aliases for these?

Reply:
nah, it was a bit more confusing with them as I couldn't really make them meaningful and 1-character.